AnkiDroid/src/main/java/com/ichi2/anki/DeckPickerFloatingActionMenu.kt

```diff
@@ -43,7 +41,7 @@ class DeckPickerFloatingActionMenu(view: View, private val deckPicker: DeckPicke
     private fun showFloatingActionMenu() {
         mLinearLayout.alpha = 0.5f
         if (mFragmented) {
-            mStudyoptionsFrame!!.alpha = 0.5f
+            mStudyOptionsFrame!!.alpha = 0.5f
         }
```

Comment:
a suggestion for null safety:
```suggestion
            mStudyOptionsFrame?.let { it.alpha = 0.5f } 
        }
```